dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -479,23 +479,23 @@ ANNO_CLASSSOURCE_JARENTRY_SCAN_EXCEPTION.useraction=See the exception and the as
 # # {4} The error message from the exception
 ANNO_CLASSSOURCE_JARENTRY_CLOSE_EXCEPTION=CWWKC0072W: [ {0} ]: The close of JAR entry [{1}] for class [{2}] in [{3}] failed. Error message: {4}
 ANNO_CLASSSOURCE_JARENTRY_CLOSE_EXCEPTION.explanation=The close of the file caused an exception.
-ANNO_CLASSSOURCE_JARENTRY_CLOSE_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_JARENTRY_CLOSE_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
```

Comment:
Change
' A hashcode id used to identify the object emitting the message'
to
' A hash code ID used to identify the object that emits the message'

Reply:
Updated.